### PR TITLE
ART-2183: Validate RHCOS before promoting a nightly

### PR DESCRIPTION
**Pull request by vfreex**
unless `BYPASS_RHCOS_VALIDATION` is checked

Requires Doozer PR https://github.com/openshift/doozer/pull/289/files.

Jenkins job is tested with https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/hack/job/yuxzhu/job/aos-cd-jobs/job/build%252Frelease/21/console.

### CODE REVIEW

File: pipeline-scripts/release.groovy [hunk 1]
```diff
@@ -130,6 +143,9 @@ def getArchPrivSuffix(arch, priv) {
     return suffix
 }
 
+def stageValidateRHCOS(from_release_tag) {
+}
+
```
Comment (sosiouxme) [inline pipeline-scripts/release.groovy:148]: did you leave this in by accident? i don't think we need a separate stage for this.

**Review by sosiouxme — CHANGES_REQUESTED** [review]
This looks great, but the `release` job has become `promote`, can you do it over there please? :)

File: pipeline-scripts/release.groovy (hunk 1 above)
Comment (vfreex) [reply pipeline-scripts/release.groovy]: Right... will remove this

**Review by vfreex — COMMENTED** [review]
PR updated

**Comment by sosiouxme** [comment]
became obsolete :-/

**Pull request closed without merging**